conf.d/main

```diff
@@ -18,13 +18,22 @@ CONF=$WEBROOT/.env
 LOC_SHARE=/usr/local/share
 LOC_BIN=/usr/local/bin
 
+#Update php related config files for ioncube loader
+sed -i '2 a zend_extension = "/usr/lib/php/20220829/ioncube_loader_lin_8.2.so"' /etc/php/8.2/apache2/php.ini
+sed -i '2 a zend_extension = "/usr/lib/php/20220829/ioncube_loader_lin_8.2.so"' /etc/php/8.2/cli/php.ini
+
+#enable php-fpm for ioncube loader
+a2enconf php8.2-fpm
+service php8.2-fpm restart
+systemctl restart apache2
```

Comment:
As noted above, services don't auto start at build time and we haven't started apache yet, so no need to restart it.

Reply:
I removed the systemctl and php-fpm related lines.